package.json

```diff
@@ -38,11 +41,20 @@
   "devDependencies": {
     "@11ty/eleventy": "^1.0.2",
     "@highlightjs/cdn-assets": "^11.6.0",
+    "@open-wc/testing": "^3.1.7",
+    "@rollup/plugin-commonjs": "^23.0.2",
+    "@rollup/plugin-replace": "^5.0.1",
     "@stackoverflow/stacks-editor": "^0.8.2",
     "@stackoverflow/stacks-icons": "^3.1.0",
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/user-event": "^14.4.3",
     "@typescript-eslint/eslint-plugin": "^5.42.0",
     "@typescript-eslint/parser": "^5.44.0",
-    "backstopjs": "^6.1.4",
+    "@web/dev-server-esbuild": "^0.3.3",
+    "@web/dev-server-rollup": "https://github.com/giamir/web/raw/fix-dev-server-rollup/packages/dev-server-rollup/web-dev-server-rollup-0.3.19.tgz",
```

Comment:
I'm ok with this as a short-term fix (until upstream releases a patched version).

No changes necessary.

Reply:
Yeah, I think so too. We are also covered from a security perspective since we have the content integrity check thanks to the `package-lock.json`. I will follow up again about my [one-liner PR](https://github.com/modernweb-dev/web/pull/2079) for WTR.